constraint_builder.h

```diff
@@ -57,6 +59,39 @@ class ConstraintBuilder {
         return velocityInRange(i, v, v);
     }
 
+    ConstraintBuilder &accGreaterEq(size_t i, constraint_t v) {
+        return accelerationInRange(i, v);
+    }
+
+    ConstraintBuilder &accLessEq(size_t i, constraint_t v) {
+        return accelerationInRange(i, NEG_INF<N_DIM>, v);
+    }
+
+    ConstraintBuilder &accEq(size_t i, constraint_t v) {
+        return accelerationInRange(i, v, v);
+    }
+
+    ConstraintBuilder &accGreaterEqFromTo(size_t a, size_t b, constraint_t v) {
+        assert(a < b);
+        while(a != b)
```

Comment:
Currently on main branch constraints velocitiesInRange, positionsInRange are defined by closed intervals, so please make it consistent (either by modifying acc constraints or the other 2 types).